sections/discussion/discussion-chapter.tex

```diff
@@ -1,9 +1,153 @@
+However, as we wanted the focus of this paper to be a broad comparison of state-of-the-art methods, we concluded that it would be beneficial to include all algorithms in all metrics.
+For the rating prediction methods, this was done by generating a predicted rating for all items for a given user, and sorting it by estimated rating and presenting the highest rated values in this list to the user.
+This approach does, however, have a minor flaw: When doing top-$N$ recommendation for a user, more than $N$ items may have the same estimated rating, in which case only some of them will be given to the user.
+This can influence the results, since we do not know whether these $N$ items are the best, or if the best results are actually in the remainder of the list.
+None the less, the methods that are taken out of their initial purpose have actually shown good results in other metrics, such as SVD performing the best in precision on the MovieLens dataset, even though it is meant for rating prediction.
+
+\subsection{Graph based methods, context-aware methods and utilizing side information}
+In the following subsection, we will try to answer the following questions: 
+\begin{itemize}
+    \item How do the context-aware methods utilize context?
+    \item Did context and side information matter?
+    \item Are timestamps good contextual variables?
+    \item How do graph-based methods compare to non-graph based methods?
+\end{itemize}
+
+\subsubsection{How do the context-aware methods utilize context?} 
+\textit{IS-UserBased} utilizes context by performing item splitting as described in \autoref{method:IS-UserBased-Graph}.
+While this item splitting approach is a simple method to model the context for items, it generates more sparsity in the data.
+Before performing the item splitting, two users could be connected through a common item, but this connection will be broken in case they did not rate the item in the same context.
+For example, if user $u_1$ rated the item $i_1$ in the context \textit{(morning, monday)} and user $u_2$ rated the same item in context \textit{(evening, monday)}, the item splitting will result in them having interacted with two different fictional items.
+To combat this sparsity, we decided to look into adding side-information to the items to re-establish this connection.
+One way to do this is by adding side-information about the movie, such as director, actors, or genre as we have seen in the KGAT method.
+We theorize that adding this side-information could help improve the recommendations generated, since it allows re-connecting these fictional items through multiple relations.
+Additionally, utilizing this along a multi-hop approach as seen in KGAT, we may be able to infer that a user has a preference for a given genre or actor in a given context, rather than just a preference for a given movie.\\
```

Comment:
```suggestion
Additionally, utilizing this along a multi-hop approach as seen in \textit{KGAT}, we may be able to infer that a user has a preference for a given genre or actor in a given context, rather than just a preference for a given movie.\\
```